### PR TITLE
Many Logic Changes

### PR DESCRIPTION
**Pull request by dyceron**
Logic changes for:
- Torvus Grove climb without Boost Ball
- Seeker Locks with Screw Attack
- Bombless Bomb Slots (some haven't been added yet)
- Lower Torvus Escape without Gravity Boost
- Transport Center Climb with Space Jump + simplifying nodes (Closes https://github.com/randovania/randovania/issues/804) (Contributes to https://github.com/randovania/randovania/issues/103)
- Doomed Entry climb with Space Jump and Screw Attack + simplifying nodes (Closes https://github.com/randovania/randovania/issues/796) (Contributes to https://github.com/randovania/randovania/issues/103)
- Quadraxis with PBs
- Hive Temple escape without Spider Ball (New nodes and updated damage requirements accordingly) (Contributes to https://github.com/randovania/randovania/issues/333) (Contributes to https://github.com/randovania/randovania/issues/103)

### CODE REVIEW

File: randovania/data/json_data/prime2.json [hunk 1]
```diff
@@ -83207,39 +82819,61 @@
...
+                                                    }
                                                 }
                                             ]
                                         }
                                     ]
+                                }
+                            }
+                        },
+                        {
+                            "name": "Event - Quadraxis Elevator Prepared",
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime2.json:82843]: Remove this node

File: randovania/data/json_data/prime2.json [hunk 2]
```diff
@@ -83207,39 +82819,61 @@
...
+                                    "type": "and",
+                                    "data": []
+                                }
+                            }
+                        },
+                        {
+                            "name": "Event - Prepare Elevator Before Fight",
+                            "heal": true,
+                            "node_type": "event",
+                            "event_index": 96,
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime2.json:82858]: Change this to event 95

File: randovania/data/json_data/prime2.json [hunk 3]
```diff
@@ -83207,39 +82819,61 @@
...
+                                    "type": "and",
+                                    "data": []
+                                }
+                            }
+                        },
+                        {
+                            "name": "Event - Prepare Elevator After Fight",
+                            "heal": true,
+                            "node_type": "event",
+                            "event_index": 97,
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime2.json:82870]: Change this to event 95

File: randovania/data/json_data/prime2.json [hunk 4]
```diff
@@ -779,6 +779,21 @@
...
                 "short_name": "Event94"
+            },
+            {
+                "index": 95,
+                "long_name": "Quadraxis Elevator Prepared",
+                "short_name": "Event95"
+            },
+            {
+                "index": 96,
+                "long_name": "Prepare Elevator Before Fight",
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime2.json:790]: Remove this event

File: randovania/data/json_data/prime2.json [hunk 5]
```diff
@@ -779,6 +779,21 @@
...
+                "short_name": "Event96"
+            },
+            {
+                "index": 97,
+                "long_name": "Prepare Elevator After Fight",
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime2.json:795]: Remove this event

File: randovania/data/json_data/prime2.txt [hunk 6]
```diff
@@ -4394,76 +4338,67 @@ Asset id: 2763180926
...
+                      Any of the following:
+                          Morph Ball Bomb and Bomb Space Jump (Normal) and Difficulty: Normal
+                          Screw Attack and Slope Jump (Easy) and Difficulty: Easy
+  > Sand Processing/Pickup (Missile)
+      After Sand Processing Sand Drained
+  > Sand Processing/Event - Sand Processing Sand Drained
+      All of the following:
+          Scan Visor and Morph Ball
+          Any of the following:
+              Boost Ball
```
[the remark this answers is not in the document]
Comment (dyceron) [reply randovania/data/json_data/prime2.txt:4389]: Oh I think I was supposed to add a wall boost here, which I believe is possible but never added. Though it should have a bomb or bombless requirement as well which I thought I added?

File: randovania/data/json_data/prime2.json (hunk 4 above)
Comment (dyceron) [reply randovania/data/json_data/prime2.json:790]: The quad elevator prepared?

File: randovania/data/json_data/prime2.json (hunk 5 above)
Comment (dyceron) [reply randovania/data/json_data/prime2.json:795]: Which one?

File: randovania/data/json_data/prime2.json [hunk 7]
```diff
@@ -83222,39 +83030,44 @@
...
                                     ]
+                                }
+                            }
+                        },
+                        {
+                            "name": "Event - Prepare Elevator Before Fight",
+                            "heal": true,
+                            "node_type": "event",
+                            "event_index": 95,
+                            "connections": {}
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime2.json:83058]: A node needs at least one connection, otherwise there's no way out

**Review by henriquegemignani — APPROVED** [review]

**Pull request merged**